### PR TITLE
Upgraded to specs2 3.0.1

### PR DESCRIPTION
**Pull request by etorreborre**

### CODE REVIEW

File: project/build.scala [hunk 1]
```diff
@@ -23,14 +23,13 @@ object build extends Build {
       , "-Xlint"
       , "-Xfatal-warnings"
       ) ++ (if (scalaBinaryVersion.value != "2.10") Seq("-Ywarn-unused-import") else Seq())
-     , scalacOptions in (Compile,console) := Seq("-language:_", "-feature")
-     , scalacOptions in (Test,console) := Seq("-language:_", "-feature")
```
Comment (markhibberd) [inline project/build.scala]: These are required so fatal-warnings doesn't kill the repl.

**Comment by markhibberd** [comment]
:+1:

**Comment by markhibberd** [comment]
(sorry :+1: after that console flags are added back)

File: project/build.scala (hunk 1 above)
Comment (etorreborre) [inline project/build.scala]: Sorry I wanted to test that and forgot to add back the lines.

**Comment by etorreborre** [comment]
@markhibberd I was going through old opened PRs. I've updated this one with the most recent version of specs2 (but also Scala). You wrote :+1: but you need to merge it since I don't have commit rights on that repo :-).

**Pull request merged**